hedera-node/src/main/java/com/hedera/services/state/migration/UniqueTokensMigrator.java

```diff
@@ -0,0 +1,70 @@
+package com.hedera.services.state.migration;
+
+/*-
+ * ‌
+ * Hedera Services Node
+ * ​
+ * Copyright (C) 2018 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.hedera.services.ServicesState;
+import com.hedera.services.state.merkle.MerkleUniqueToken;
+import com.hedera.services.state.virtual.UniqueTokenKey;
+import com.hedera.services.state.virtual.UniqueTokenValue;
+import com.hedera.services.state.virtual.VirtualMapFactory;
+import com.hedera.services.utils.EntityNumPair;
+import com.swirlds.jasperdb.JasperDbBuilder;
+import com.swirlds.merkle.map.MerkleMap;
+import com.swirlds.virtualmap.VirtualMap;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static com.hedera.services.utils.MiscUtils.forEach;
+
+public class UniqueTokensMigrator {
+	public static final int TARGET_RELEASE = StateVersions.RELEASE_0270_VERSION;
```

Comment:
Note: This needs to be updated to reflect the release version we are targetting. 0280 ?

Reply:
removed references to version.